Alterações3

MichaelTomasMuane opened the pull request.

MichaelTomasMuane committed:
Team, Versão 1

Adicionando pagina do teams ao html., adicionado um video para our team, adicionado footer, retirado letras pequenas do index (descrição).

MichaelTomasMuane committed:
changes

Lone1wolf commented:
Changes add, team.html added yet not finished, only anded video (Our Team) and footer., add video to assets and linked the Google form to reserve a place on index.html.

MichaelTomasMuane committed:
Add team final V1

Adicionado team para porte de pc, recureparado imagens da pasta imagens, adicionado títulos mais instrutivos sobre a pagina.
